vuln/core/11.json

```diff
@@ -0,0 +1,9 @@
+{
+  "cve": [],
+  "vulnerable": "6.x",
+  "patched": "^6.9.0",
+  "ref": "https://nodejs.org/en/blog/vulnerability/october-2016-security-releases/ for",
+  "author": "Jann Horn",
```

Comment:
Ref https://github.com/nodejs/security-wg/pull/35#discussion_r130684626, changing `author` to something else seems like a good idea.

Maybe `reporter`?